src/MongoDB/Session.c

```diff
@@ -390,6 +390,24 @@ static PHP_METHOD(Session, endSession)
 	mongoc_client_session_destroy(intern->client_session);
 } /* }}} */
 
+/* {{{ proto void MongoDB\Driver\Session::isInTransaction(void)
+   Returns whether a multi-document transaction is in progress */
+static PHP_METHOD(Session, isInTransaction)
```

Comment:
This method name looks strange to me, but I think it's preferable to the more verbose option of `isTransactionInProgress()` or the like.

Reply:
Change it to ``inTransaction`` ? Don't like that much either...

Reply:
I didn't want to suggest `inTransaction` because it's not consistent with prefixing boolean-returning methods with "is". Totally OK with sticking with `isInTransaction()`.